source/MRViewer/MRUIStyle.h

```diff
@@ -172,41 +172,30 @@ namespace detail
     // `UI::drag()` uses this internally to draw tooltips.
     // Pass `getDragRangeTooltip()` as the parameter.
     MRVIEWER_API void drawDragTooltip( std::string rangeText );
+
+    // Wraps `ImGui::MarkItemEdited()`, to avoid including `imgui_internal.h`.
+    MRVIEWER_API void markItemEdited( ImGuiID id );
```

Comment:
why is it needed on user end?

Reply:
This is in `namespace detail`. I only need it internally in tamplate.